doc/en/components/grids/_shared/cell-editing.md

```diff
@@ -1193,7 +1229,7 @@ Here, we are validating two columns. If the user tries to change an artist's **D
 <!-- React -->
 <!-- ComponentStart: HierarchicalGrid -->
 ```tsx
-public handleCellEdit(sender: IgrHierarchicalGrid, event: IgrGridEditEventArgs): void {
+public handleCellEdit(event: IgrGridEditEventArgs): void {
```

Comment:
The sample and snippet for the hierarchical grid don't seem to match since they show different data.